bin/bootstrap

```diff
@@ -50,6 +50,8 @@ brew services restart caddy
 # dnsmasq handles the local dns translation from *.lessonly.test -> 127.0.0.1
 # install dnsmasq config
 cp bin/dnsmasq.conf $(brew --prefix)/etc/dnsmasq.conf
+# make /etc/resolver directory if it does not exist
+sudo mkdir -p /etc/resolver
```

Comment:
`mkdir -p` only makes the directory if it doesn't exist already. I believe that _maybe_ dnsmasq should be creating this folder automatically but, when running lldevconfig on my M1 machine, it did not. So manually creating the directory solved my issues.